dashboard/modules/reporter/tests/test_reporter.py

```diff
@@ -840,6 +840,51 @@ def verify():
     wait_for_condition(verify, timeout=20)
 
 
+@pytest.mark.skipif(
+    os.environ.get("RAY_MINIMAL") == "1",
+    reason="This test is not supposed to work for minimal installation.",
+)
+@pytest.mark.skipif(sys.platform == "win32", reason="No memray on Windows.")
+def test_get_memory_profile_running_task(shutdown_only):
+    """
+    Verify that we throw an error for a non-running task.
+
+    """
```

Comment:
This docstring doesn't match the test behavior. Instead, it looks like we're verifying that we get a success response when the task is running?